ProteGO/Common/Extensions/UITextField/UITextFireld+Factory.swift

```diff
@@ -10,20 +10,20 @@ extension UITextField {
 
         let placeholderAttributes: [NSAttributedString.Key: Any] = [
             .font: Fonts.poppinsMedium(16).font,
-            .foregroundColor: Colors.greyish]
+            .foregroundColor: UIColor(asset: Assets.greyish)!] // swiftlint:disable:this force_unwrapping
         textField.attributedPlaceholder = NSAttributedString(string: placeholder, attributes: placeholderAttributes)
 
         let textAttributes: [NSAttributedString.Key: Any] = [
             .font: Fonts.poppinsMedium(16).font,
-            .foregroundColor: Colors.bluishGreen]
+            .foregroundColor: UIColor(asset: Assets.bluishGreen)!] // swiftlint:disable:this force_unwrapping
```

Comment:
dodawanie swiftlint:disable przy kazdym uzyciu tego skrótu z kolorami to kiepski pomysł IMO.

Dałoby radę jakos ustawić swiftlinta aby to on to odwrapowywał jakos?

Reply:
Nie przy każdym, inne wywołania tego nie mają. Tu pojawia się problem że Dictionary jest typu `[NSAttributedString.Key: Any]`. Musielibyśmy ustawić to na `[NSAttributedString.Key: Any?]` ale wtedy `NSAttributedString` tego nie łyknie, bo jego publiczny inicajalizator to:
`public init(string str: String, attributes attrs: [NSAttributedString.Key : Any]? = nil)`.

SwiftGen jest tylko wrapper'em na `UIColor.init?(named name: String)` a koloroczywiście może nie istnieć *.xctasset.

Nie sprawdzałem czy swiftlint można tak ustawić, ale wątpię? Poprzednia implementacja też używała force unwrap'ów i też był disabled state.  Można by wyłączyć dla całego pliku teraz. Ewentualnie zastosowanie `guard let`

Reply:
ok, masz rację - jakos umkeło mi to ze przypadek szczególny.

Daje approve